template-parts/content-grid.php

```diff
@@ -40,7 +40,7 @@
 	</header><!-- .entry-header -->
 
 	<?php if ( 'post' === get_post_type() ) : ?>
-		<div class="entry-meta">
+		<div class="entry-meta" <?php echo $rtl_css_override; ?>>
```

Comment:
Couldn't this be done in CSS if needed?

Reply:
Yeah, makes sense @goldenapples as all `.entry-meta` are affected. Fixing it now.

Reply:
@goldenapples I noticed that we can't add the CSS to `style.css` because when `style-rtl.css` gets automatically generated the values would get reversed. Are you ok with this solution?

Reply:
Re: your discussion above,

> I noticed that we can't add the CSS to style.css because when style-rtl.css gets automatically generated the values would get reversed. Are you ok with this solution?

it should be possible to add this to the SCSS files by using an [rtlcss Control Directive](https://rtlcss.com/learn/usage-guide/control-directives/) to ensure it won't be reversed or stripped out. You can scope the rule to only apply when RTL is active by nesting within `body.rtl`, e.g.,

```css
/*rtl:ignore*/
body.rtl .entry-meta {
    direction: ltr;
    text-align: right;
}
```

Note that no CSS should be added to `style.css` itself, only to `sass/style.scss`. `style.css` is _also_ auto-generated.